README.md

```diff
@@ -1,2 +1,23 @@
 # lobster-ice
 STEP 2020 Capstone Project :) 
+
+### Step 1: Start Server
```

Comment:
Please add some explanation what the component is.
Better names suggested for steps: 
Start Server -> Start Service Api Node on port 8080
Web Application -> Start React Ui Node on port 3000
Chrome Extension -> Install the extension in developer mode. Please point out which is the root folder of the extension.

Reply:
Thank you for this!